test/lib/jdk/test/lib/containers/docker/DockerTestUtils.java

```diff
@@ -120,59 +120,64 @@ private static boolean isDockerEngineAvailableCheck() throws Exception {
         return true;
     }
 
+     /**
+     * Build a container image that contains JDK under test.
+     * The jdk will be placed under the "/jdk/" folder inside the image/container file system.
+     *
+     * @param imageName name of the image to be created, including version tag
+     * @throws Exception
+     */
+    public static void buildJdkContainerImage(String imageName) throws Exception {
+        buildJdkContainerImage(imageName, null);
+    }
 
-    /**
-     * Build a docker image that contains JDK under test.
-     * The jdk will be placed under the "/jdk/" folder inside the docker file system.
+     /**
+     * Build a container image that contains JDK under test.
+     * The jdk will be placed under the "/jdk/" folder inside the image/container file system.
      *
-     * @param imageName     name of the image to be created, including version tag
-     * @param dockerfile    name of the dockerfile residing in the test source;
-     *                      we check for a platform specific dockerfile as well
-     *                      and use this one in case it exists
-     * @param buildDirName  name of the docker build/staging directory, which will
-     *                      be created in the jtreg's scratch folder
+     * @param imageName         name of the image to be created, including version tag
+     * @param dockerfileContent content of the Dockerfile; use null to generate default content
      * @throws Exception
      */
-    public static void
-        buildJdkDockerImage(String imageName, String dockerfile, String buildDirName)
-            throws Exception {
+    public static void buildJdkContainerImage(String imageName, String dockerfileContent) throws Exception {
 
-        Path buildDir = Paths.get(".", buildDirName);
+        // Create an image build/staging directory
+        Path buildDir = Paths.get(".", "image-build");
```

Comment:
to make it more robust and possible to have more than one container within a test, I'd use `imagName` + "image-build" as build dir name.

Reply:
you also don't need to have `.` as a first argument, `Paths.get("x")` returns you a path to "x" in current directory.